resources/views/layouts/right_vertical_transparent.blade.php

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+
+<html lang="{{ app()->getLocale() }}" dir="{{ backpack_theme_config('html_direction') }}">
+
+<head>
+    @include(backpack_view('inc.head'))
+</head>
+
+<body class="o-auto {{ backpack_theme_config('classes.body') }}">
+
+@include(backpack_view('layouts.partials.light-dark-mode'))
+
+<div class="page">
+
+    @include(backpack_view('layouts._right_vertical_transparent.menu_container'))
+
+    <div class="page-wrapper">
+
+        <div class="@if(backpack_theme_config('options.useStickyHeader')) sticky-top @endif">
+            @include(backpack_view('layouts._right_vertical_transparent.header_container'))
+        </div>
+
+        <div class="page-body">
+            <main class="{{ backpack_theme_config('options.useFluidContainers') ? 'container-fluid' : 'container-xl' }}">
+
+                @yield('before_breadcrumbs_widgets')
+                @includeWhen(isset($breadcrumbs), backpack_view('inc.breadcrumbs'))
+                @yield('after_breadcrumbs_widgets')
+                @yield('header')
+
+                <div class="container-fluid animated fadeIn">
+                    @yield('before_content_widgets')
+                    @yield('content')
+                    @yield('after_content_widgets')
+                </div>
+            </main>
+        </div>
+
+        @include(backpack_view('inc.footer'))
```

Comment:
I don't think the footer is important enough (visually) to make it worth being white. So how about we make it transparent, in all layouts? 

<img width="1512" alt="CleanShot 2023-04-19 at 12 17 36@2x" src="https://user-images.githubusercontent.com/1032474/233029293-b7a8eba4-37b5-40bd-9c2b-e6939dfc9444.png">

vs

<img width="1512" alt="CleanShot 2023-04-19 at 12 17 50@2x" src="https://user-images.githubusercontent.com/1032474/233029340-0d3a26f9-4746-4bf6-8426-5b747ac5bbbd.png">